pkg/kubernetes/run.go

```diff
@@ -20,7 +20,7 @@ type RunOptions struct {
 }
 
 const (
-	PluginProtocolVersion uint = 1
+	PluginProtocolVersion = 1
```

Comment:
Shoot, when the secrets plugin protocol was changed in Porter we should have also changed the protocol version to 2 in Porter. I'll fix that now in Porter and then before this is merged, it should be updated to match.

Since we are already referencing the Porter packages, we can use the constant defined in porter to help keep these values in sync.

Reply:
OK this is in the release/v1 branch now.